src/App.js

```diff
@@ -1,23 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='App'>
```

Comment:
if we chose to go for TitleCase for class name, let's keep that standard thorughout. I normally used to do lower case for class name but as long as we keep standard  that's cool :)

Reply:
it's a default class name by react, let's follow this naming standard.